doc/sphinxman/source/scf.rst

```diff
@@ -739,6 +739,21 @@ To avoid this, either set |scf__df_basis_scf| to an auxiliary
 basis set defined for all atoms in the system, or set |scf__df_scf_guess|
 to false, which disables this acceleration entirely.
 
+Linear-Scaling Exchange Build
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Methods that build K matrices in linear time have been developed. One of which is the Linear Exchange (LinK) Method 
+to build K matrices, described in [Ochsenfeld:1998:1663]_. The LinK method is available to use as part
+of the Direct SCF algorithm. LinK offers a fast alternative to density-fitted K builds when the amount of available memory is low, 
+especially when combined with Density-matrix based ERI screening and incremental Fock builds.
+
+To turn on and control the LinK algorithm, here are the list of options provided.
+
+  |scf__do_linK|: Defaults to false. If turned on, the K matrix will be built using the algorithm described in [Ochsenfeld:1998:1663]_.
+
+  |scf__linK_ints_tolerance|: Defaults to 1.0e-12. The integral screening tolerance used for sparsity-prep in the LinK algorithm. Note that
+    this option is separate from and independent of the |scf__ints_tolerance| option. A tighter value is recommended for more diffuse basis sets (1.0e-14).
```

Comment:
Maybe a stupid question, but why is this different from `scf_ints_tolerance`?

Reply:
I made it separate in case we want to control the error of LinK separately from the error of Schwarz or Density Screening. This is an expert option and should default to whatever the value of `scf__ints_tolerance` is.